firmware/src/mPMTTestStand.cpp

```diff
@@ -17,11 +17,13 @@ mPMTTestStand::mPMTTestStand(const mPMTTestStandIO& io) :
     thermistor_optical(io.pin_therm_optical)
 {
     // Nothing else to do
```

Comment:
is this just to initialize each thermistor?

Reply:
Yes, like to tell each thermistor it's pin number. Is that what you're asking about?